tests/tools/onert_train/src/nnfw_util.h

```diff
@@ -36,6 +36,9 @@ namespace onert_train
 uint64_t num_elems(const nnfw_tensorinfo *ti);
 uint64_t bufsize_for(const nnfw_tensorinfo *ti);
 
+std::string toString(NNFW_TRAIN_OPTIMIZER opt);
+std::string toString(NNFW_TRAIN_LOSS loss);
+std::string toString(NNFW_TRAIN_LOSS_REDUCTION loss_rdt);
```

Comment:
I have a plan to use this function later, here : 

 https://github.com/Samsung/ONE/pull/12445#r1466143798